server/api.go

```diff
@@ -294,6 +294,13 @@ func AddDefaultAccess(a *knox.Access) {
 	defaultAccess = append(defaultAccess, *a)
 }
 
+var accessCallback func(knox.AccessCallbackInput) (bool, error)
```

Comment:
I think you need to create an initial implementation here, otherwise there will be some issues when calling the callback if you don't call the SetAccesssCallback function.

Reply:
We already do this in authorizeRequest by checking if it's nil (func default zero property), and returning false if so